### PR TITLE
refactor(sass): improve all components (S01E01) - INNO-1020

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Update scss file for all component, according to documentation.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

File: src/systems/ec/ec-component/ec-component-accordion/ec-component-accordion.scss [hunk 1]
```diff
@@ -2,52 +2,12 @@
...
-  margin: 0;
-  padding: map-get($ecl-spacing, 's');
-
-  @include ecl-media-breakpoint-up(md) {
-    padding-left: map-get($ecl-spacing, 'xxl');
-  }
-}
+// Import base and generic
+@import '@ecl/ec-base/ec-base';
+@import '@ecl/generic-component-accordion/generic-component-accordion';
```
Comment (yhuard) [inline src/systems/ec/ec-component/ec-component-accordion/ec-component-accordion.scss]: Missing check on `ecl-icon`

File: src/systems/ec/ec-component/ec-component-banner/ec-component-banner.scss [hunk 2]
```diff
@@ -3,6 +3,16 @@
...
  */
 
+// Import base and generic
+@import '@ecl/ec-base/ec-base';
 @import '@ecl/generic-component-banner/generic-component-banner';
 
-@include ecl-banner();
+// Check if overridden dependencies are already loaded, if needed
+@include check-imports(
+  ('ec-style-typography-heading', 'ec-style-typography-paragraph')
```
Comment (yhuard) [inline src/systems/ec/ec-component/ec-component-banner/ec-component-banner.scss]: Missing check on `ecl-image` (`ec-style-image`i guess)

**Review by yhuard — CHANGES_REQUESTED** [review]
A few missing checks (in both EC and EU systems) + please update the breadcrumb in `website/index.html`

File: src/systems/ec/ec-component/ec-component-banner/ec-component-banner.scss (hunk 2 above)
Comment (emeryro) [reply src/systems/ec/ec-component/ec-component-banner/ec-component-banner.scss]: I don't see icons in banner

**Review by yhuard — APPROVED** [review]

**Review by yhuard — CHANGES_REQUESTED** [review]
Please update the tests as well (see https://github.com/ec-europa/europa-component-library/blob/2ff536acbe0e0306f33cdf962be7904212b9604f/src/systems/ec/ec-component/ec-component-breadcrumb/test/spec/breadcrumbs.js#L23)

**Review by yhuard — APPROVED** [review]

**Pull request merged**